lib/elixir/lib/io/ansi/docs.ex

```diff
@@ -291,6 +291,8 @@ defmodule IO.ANSI.Docs do
     col =
       col
       |> String.replace("\\\|", "|")
+      |> String.replace(~r/-+/, "-")
```

Comment:
Oh, I completely spaced out on the `+`. We cannot use regexes because it breaks precompiled Elixir on OTP 20. The few cases we do we need to call `Regex.recompile`. Ideally we want to find a way to write this that does not rely on regexes.